instrumentation/opentelemetry-instrumentation-openai/pyproject.toml

```diff
@@ -0,0 +1,53 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "opentelemetry-instrumentation-openai"
```

Comment:
Unfortunately this is already taken and we have to sort out how to handle that

Reply:
The convention for naming would suggest this name unfortunately :sweat:

Reply:
I suggest `opentelemetry-instrumentation-openai-v2` and the following plan:
- we'll start with this name and evolve it along with semconv to have feature parity with [`opentelemetry-instrumentation-openai`](https://github.com/traceloop/openllmetry/tree/main/packages/opentelemetry-instrumentation-openai)
- we'll need to check with @nirga, but based on the previous discussions they should, at some point, be able to publish `opentelemetry-instrumentation-openai` package from OTel
- when Traceloop and we are ready, we'll start publishing `opentelemetry-instrumentation-openai` v2 from OTel and stop publishing `opentelemetry-instrumentation-openai-v2`.

This would let us unblock this work and will give traceloop time to migrate. All the details need further discussion (and of course name I came up with is almost random - any other suggestions are welcome).